pkg/bazel/bazel.go

```diff
@@ -13,12 +13,56 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"strings"
+	"time"
 
+	"aspect.build/cli/pkg/logger"
 	"github.com/bazelbuild/bazelisk/core"
 	"github.com/bazelbuild/bazelisk/repositories"
 	"google.golang.org/protobuf/proto"
 )
 
+var invocationID string = ""
+
+func generateInvocationID() {
+	t := time.Now()
+
+	dateString := stringifyInt(t.Year(), 4) + stringifyInt(int(t.Month()), 2) + stringifyInt(t.Day(), 2)
+
+	nanosecondStr := stringifyInt(t.Nanosecond(), 6)
+
+	timeString := stringifyInt(t.Hour(), 2) +
+		stringifyInt(t.Minute(), 2) +
+		"-" +
+		stringifyInt(t.Second(), 2) +
+		nanosecondStr[:2] +
+		"-" +
+		nanosecondStr[2:6]
+
+	superSpecialHexString := "617370656374" // I wonder what this says when converted back to a string?
+
+	invocationID = dateString + "-" + timeString + "-" + superSpecialHexString
```

Comment:
Did we think about putting the process-id in here?

Reply:
I dont think so? The discussion we had was mostly about how we should set the ID and make it based on a date so that they are ordered

Reply:
I'd say it could be useful, especially if aspect is is launched by another tool which records PIDs, or if aspect launches child processes that may log their parent ID etc. Then you can actually find the aspect cli logs...

Reply:
I took a quick look. Looks like I can get the current PID easily so shouldnt be a problem. The bigger concern is that there are rules about the invocation ID we give. I would add the PID at the end. After the second `-`. This bit needs to be 12 characters long. Based on some googling I dont think we have an issue of it being longer than that, but it will always be shorter. So if we had a PID of `12345` I would likely want our invocation ID to end with `000000012345`. Thoughts?

Reply:
I assume that's fine. Are we following a standard of some type here?

Reply:
Cool. And its just the standard UUID format. Bazel will reject it if we dont follow it correctly. The standard format is `8-4-4-4-12`. Where each number represents the expected number of hex characters.